var/spack/repos/builtin/packages/rivet/package.py

```diff
@@ -264,24 +49,20 @@ class Rivet(AutotoolsPackage):
 
     # The following versions were not a part of LCG stack
     # and thus the exact version of YODA is unknown
-    depends_on("yoda@1.7.0:1.7", when="@2.6.0,2.7.0,2.7.1,3.0.0,3.0.2")
-    depends_on("yoda@1.5.0:1.5", when="@2.4.1")
+    depends_on("yoda@1.7.0:1.7", when="@3.0.0,3.0.2")
 
     depends_on("hepmc", when="hepmc=2")
     depends_on("hepmc3", when="hepmc=3")
-    depends_on("boost", when="@:2.5.0")
     # TODO: replace this with an explicit list of components of Boost,
     # for instance depends_on('boost +filesystem')
     # See https://github.com/spack/spack/pull/22303 for reference
-    depends_on(Boost.with_default_variants, when="@:2.5.0")
+    depends_on(Boost.with_default_variants)
```

Comment:
https://gitlab.com/hepcedar/rivet/-/releases/rivet-2.5.0: there is not dependency on boost anymore.

Reply:
"3. Elimination of the Boost library in favour of C++11."

Reply:
Deleted!